### PR TITLE
Add all data samples provided in R and update `load_data`

### PR DESCRIPTION
**Pull request by iuliadmtru**
`load_data` now works for general file names. I added all the datasets provided by the R package, downloaded from here: https://github.com/cran/survey/tree/master/data and converted to CSV.

### CODE REVIEW

File: src/example.jl [hunk 1]
```diff
@@ -11,14 +11,19 @@ being the name of the dataset.
...
 The API program has been discontinued at the end of 2018. Information is archived at https:
 //www.cde.ca.gov/re/pr/api.asp
 """
 function load_data(name)
-    @assert name ∈ ["apiclus1", "apiclus2", "apipop", "apistrat", "apisrs"]
-    CSV.read(asset_path(name * ".csv"), DataFrame, missingstring="NA")
+	name = name * ".csv"
+	path_len = length(pathof(Survey))
+	termination_len = length("src/Survey.jl")
+	assets_path = pathof(Survey)[1:path_len - termination_len] * "assets"
```
Comment (ayushpatnaikgit) [inline src/example.jl:25]:
```suggestion
	assets_path = joinpath(PKG_DIR, "assets")
```

Comment (ayushpatnaikgit) [inline src/example.jl:25]:
```suggestion
	assets_path = joinpath(PKG_DIR, "assets")
```

Comment (ayushpatnaikgit) [reply src/example.jl:25]: I think tab spaces in this file are inconsistent.

**Comment by ayushpatnaikgit** [comment]
We also need a separate documentation file, perhaps called “Sample datasets”. 
This should contain documentation about all the sample datasets.

**Comment by ayushpatnaikgit** [comment]
Perhaps, we should 
1) Change the name of the folder from `assets` to `sample_datasets`. This seems apt since we are reading all the `.csv` files from the folder, and that means we won't have another other kinds of assets. 
2) Change the filename containing the `load_dataset` function from `example.jl` to `sample_datasets.jl` or `load_datasets.jl`. 

@greimel what do you think?

**Comment by greimel** [comment]
Have you checked that it is permitted to make these files available here?

**Comment by ayushpatnaikgit** [comment]
> Have you checked that it is permitted to make these files available here?

Should be fine since we are using the same license as the R package that contains these files. But I'll verify this with Thomas Lumley.

**Review by ayushpatnaikgit — APPROVED** [review]

**Comment by ayushpatnaikgit** [comment]
> Perhaps, we should
> 
>     1. Change the name of the folder from `assets` to `sample_datasets`. This seems apt since we are reading all the `.csv` files from the folder, and that means we won't have another other kinds of assets.
> 
>     2. Change the filename containing the `load_dataset` function from `example.jl` to `sample_datasets.jl` or `load_datasets.jl`.
> 
> 
> @greimel what do you think?

@iuliadmtru can you make these changes in a new PR?

**Pull request merged**